src/SConscript

```diff
@@ -71,6 +71,9 @@ def scons():
 
     Export('daos_version', 'API_VERSION')
 
+    # For Common library and headers.
+    env.Install(os.path.join('$PREFIX', 'include/daos'), 'include/daos/tse.h')
```

Comment:
good catch. not sure why this was removed.
sigh we should add a 2.4.1 backport which means we need rc2 it seems.

Reply:
Apologies for that, the intention wasn't to remove a header.  In fact, I thought if you removed a header from the install include path then the rpm failed to build unless you updated the spec file?

The change this partially reverts isn't on the 2.4 branch.

Reply:
ok false alarm. this looks like it was not backported to 2.4.

Reply:
the issue is that we do not rebuild the daos-VOL and just use pre-installed RPMs in CI. so it would not be caught if some change is introduced that breaks the source build.

Reply:
My point was that if the install path doesn't match what's in the rpm specfile then the build will fail so adding or removing a header from the RPM requires an update to both scons and the specfile, if you change one without the other then it'll fail either way.  The daos rpm build should have caught it, not daos-VOL testing.

Reply:
i see. yea misunderstood.